apps/app/src/pages/Build/ScreenshotsDiffCard.js

```diff
@@ -50,7 +54,11 @@ function Screenshot({ screenshot, title }) {
   );
 }
 
-export function ScreenshotsDiffCard({ screenshotDiff, open, ...props }) {
+export function ScreenshotsDiffCard({
+  screenshotDiff,
+  open = "true",
```

Comment:
```suggestion
  opened = true,
```

Not sure, but looks weird with string. Also should be `opened`.